test/fs.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const metatests = require('metatests');
+const fs = require('fs');
+const { mkdirp } = require('..');
+const path = require('path');
+
+const beforeEach = dir => {
```

Comment:
It should be `test.beforeEach` and `test.afterEach`. They will be called before and after each test, so you don't need different directories for each test like 'test/ex1/ex2' and 'test/1/2', it will be only one of them.